VagrantProvision.sh

```diff
@@ -193,89 +254,90 @@ PATH=$HOOT_HOME/bin:$PATH
 EOT
 fi
 
-# Change Tomcat umask to group write
 if ! grep -i --quiet 'umask 002' /etc/default/tomcat6; then
-echo "Changing Tomcat umask to group write"
+echo "Changing Tomcat umask to group write..."
 sudo bash -c "cat >> /etc/default/tomcat6" <<EOT
 # Set tomcat6 umask to group write because all files in shared folder are owned by vagrant
 umask 002
 EOT
 fi
 
-# Change Tomcat java opts
 if grep -i --quiet '^JAVA_OPTS=.*\-Xmx128m' /etc/default/tomcat6; then
-    echo "Changing Tomcat java opts"
+    echo "Changing Tomcat java opts..."
     sudo sed -i.bak "s@\-Xmx128m@\-Xms512m \-Xmx2048m \-XX:PermSize=512m \-XX:MaxPermSize=4096m@" /etc/default/tomcat6
 fi
 
-# Fix env var path for GDAL_DATA
 if grep -i --quiet 'gdal/1.10' /etc/default/tomcat6; then
-    echo "Fixing Tomcat GDAL_DATA env var path"
+    echo "Fixing Tomcat GDAL_DATA env var path..."
     sudo sed -i.bak s@^GDAL_DATA=.*@GDAL_DATA=\/usr\/local\/share\/gdal@ /etc/default/tomcat6
 fi
 
 # Remove gdal libs installed by libgdal-dev that interfere with
 # renderdb-export-server using gdal libs compiled from source (fgdb support)
-if [ -f /usr/lib/libgdal.* ]; then
-    echo "Removing GDAL libs installed by libgdal-dev"
+if [ -f "/usr/lib/libgdal.*" ]; then
+    echo "Removing GDAL libs installed by libgdal-dev..."
     sudo rm /usr/lib/libgdal.*
 fi
 
-# Create Tomcat context path for tile images
+mkdir -p $HOOT_HOME/ingest/processed
+chown -R vagrant:tomcat6 $HOOT_HOME/ingest
 if ! grep -i --quiet 'ingest/processed' /etc/tomcat6/server.xml; then
-    echo "Adding Tomcat context path for tile images"
+    echo "Adding Tomcat context path for tile images..."
     sudo sed -i.bak "s@<\/Host>@  <Context docBase=\"\/home\/vagrant\/hoot\/ingest\/processed\" path=\"\/static\" \/>\n      &@" /etc/tomcat6/server.xml
 fi
 
-# Allow linking in Tomcat context
 if ! grep -i --quiet 'allowLinking="true"' /etc/tomcat6/context.xml; then
-    echo "Set allowLinking to true in Tomcat context"
+    echo "Set allowLinking to true in Tomcat context..."
     sudo sed -i.bak "s@^<Context>@<Context allowLinking=\"true\">@" /etc/tomcat6/context.xml
 fi
 
-# Create directory for webapp
-if [ ! -d /usr/share/tomcat6/.deegree ]; then
-    echo "Creating directory for webapp"
-    sudo mkdir /usr/share/tomcat6/.deegree
-    sudo chown tomcat6:tomcat6 /usr/share/tomcat6/.deegree
-fi
-
-# Tweak dev environment to make tests run faster
-if [ ! -f $HOOT_HOME/hoot-services/src/main/resources/conf/local.conf ]; then
-    echo 'testJobStatusPollerTimeout=250' > $HOOT_HOME/hoot-services/src/main/resources/conf/local.conf
```

Comment:
$HOOT_HOME/conf/Config.sh, executed by the Tomcat deploy script, writes 'testJobStatusPollerTimeout=250' now by default to the local.conf file.  So, I removed this line.